src/_scss/pages/agencyLanding/table/_table.scss

```diff
@@ -1,6 +1,9 @@
 .agency-landing-results-table {
     display: block;
-    overflow: scroll;   
+
+	@media (max-width: $large-screen) {
+		overflow-x: scroll;
```

Comment:
this could be overflow-x: auto so the scroll bar only appears when there is scroll.  same for all 4 landing pages.

Reply:
overflow-x has been updated to auto on all four landing pages.